cocos/ui/UIEditBox/UIEditBox.h

```diff
@@ -407,7 +407,20 @@ namespace ui {
          * @param returnType One of the EditBox::KeyboardReturnType constants.
          */
         void setReturnType(EditBox::KeyboardReturnType returnType);
-            
+        
+        /**
+         * Set the inner padding in edit box.
+         * @param padding left, top, right, bottom.
+         */
+        void setPadding(float left, float top, float right, float bottom);
+        
+        /**
+         * Set the input text horizontal alignment and vertical alignment.
+         * @param hAlign the alignment for horizontally.
+         * @param vAlign the alignment for vertically.
+         */
+        void setTextAlignment(TextHAlignment hAlign, TextVAlignment vAlign);
```

Comment:
@Mazyod  @zilongshanren  I also think to separate the API, But there are one issue in Android. There are no any API is there for alignment. You set the gravity to achieve the alignment feature. There are not separate method to set gravity horizontal or vertical. Either you set single or with together. If you set only Horizontal or vertical Gravity at a time then you missed the previous set gravity.